src/documents/tutorial-userInterfaceOptions/UserInterfaceOptions.md

```diff
@@ -37,20 +37,27 @@ The rest of this tutorial will explain each of these steps in detail.
 
-1. Get the current source code from github as a [ZIP file]( https://github.com/fluid-project/infusion/archive/master.zip)
-2. Unpack the zip file you just downloaded and cd into the "infusion-master" folder that results.
-3. If necessary install [Node.js](http://nodejs.org/download/) and [Grunt](http://gruntjs.com/) `npm install -g grunt-cli`.
-4. Make your own custom build by running the `grunt` command in the Terminal. See the [README.md](https://github.com/fluid-project/infusion/blob/master/README.md#how-do-i-create-an-infusion-package) file for instructions on how to make a custom build of Infusion.
-5. The grunt command will create a zip file in the products folder. Unzip that file and move the resulting `infusion` folder somewhere convenient for your development purposes, likely in a `lib` folder in your site hierarchy.
-6. This `infusion` will include a single file containing all of the JavaScript you need: `infusion-custom.js`. You will link to this file in the headers of your HTML files.
+1. Download the current source code from Github as a [ZIP file]( https://github.com/fluid-project/infusion/archive/master.zip).
+2. Unzip the downloaded ZIP file. This will result in a new directory `infusion-master`. 
+3. Using a command line terminal, change to the `infusion-master` directory.
+4. Install [Node.js](http://nodejs.org/download/), if it hasn't been installed.
+5. Install [Grunt](http://gruntjs.com/getting-started) globally, if it hasn't been installed. For Windows, you can run `npm install -g grunt-cli` as Administrator. For Mac OSX, you can run `sudo npm install -g grunt-cli` from the command line terminal.
+6. Create your build by running the `grunt` command in the command line terminal. If you would like to customize your build, you can create a custom build by running the `grunt custom` command in the command line terminal. See the [README.md](https://github.com/fluid-project/infusion/blob/master/README.md#how-do-i-create-an-infusion-package) file for instructions on how to make a custom build of Infusion.
+7. A ZIP file, such as infusion-2.0.0.zip, will be created in the infusion-master/products directory. Copy this Infusion ZIP file into your project directory. 
+8. Now unzip the Infusion ZIP file in your project directory. This will create a new directory called infusion which contains all the Javascript and files you will need to use UI Options.
+9. Now that infusion is in your project directory, you can go ahead and clean up the files and directories we created so far. You can safely delete: 
+   * The ZIP file downloaded in Step 1.
+   * The infusion-master directory created in Step 2.
+   * The infusion-2.0.0.zip (or similar name) from your project directory.
+10. Your `infusion` folder will include a single file containing all of the JavaScript you need: `infusion-all.js`. If you made a custom build, the file is named `infusion-custom.js`. You will later link to this file in the headers of your HTML files.
```

Comment:
"You will later link to this file in the headers of your HTML files." should be something like "You will later link to this file in the `<head>` of your HTML files.`